modules/microshift-nw-ipv6-single-stack-config.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-ipv6-single-stack-config_{context}"]
+= Configuring IPv6 single-stack
+
+You can use the IPv6 network protocol by updating the {microshift-short} service configuration file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+//if you have multus, would you also use the same procedure?
+
+//* The cluster node has IPv6 addresses.
+//also does not need to be pre-config, because we are doing that here, correct?
```

Comment:
Not sure what you mean by pre-config? The host needs to have IPv6 networking ready: an IPv6 address and IPv6 routes (including the default).